query/security/common/src/main/java/org/codice/ddf/admin/security/common/fields/wcpm/Realm.java

```diff
@@ -55,12 +55,8 @@ public Realm(ServiceReader serviceReader, EnumValue<String> value) {
 
     @Override
     public List<EnumValue<String>> getEnumValues() {
-        Set<EnumValuePoller> realms = serviceReader.getServices(EnumValuePoller.class, REALM_POLLER_FILTER);
-
-        return realms.stream()
-                .findFirst()
-                .map(EnumValuePoller::getEnumValues)
-                .orElse(new ArrayList<>());
+        Iterator<EnumValuePoller> poller = serviceReader.getServices(EnumValuePoller.class, REALM_POLLER_FILTER).iterator();
```

Comment:
I found this easier to debug. There were problems with the `enumValuePoller` values not coming up properly. The exception being thrown by the stream hid the under lying issue.